core/src/test/scala/SignatureSpec.scala

```diff
@@ -28,8 +28,8 @@ class SignatureSpec extends FlatSpec
   behavior of "Signature"
   
   it should "be able to wrap a response type" in {
-    Signature("foo", "bar", List(), "baz").wrapResponse should be ("Response[baz]")
-    Signature("foo", "bar", List("baz"), "qux").wrapResponse should be ("baz => Response[qux]")
-    Signature("foo", "bar", List("baz", "qux"), "zod").wrapResponse should be ("baz,qux => Response[zod]")
+    Signature("foo",List(), Field("baz", "Baz")).wrapResponse should be ("Response[Baz]")
+    Signature("foo", List(Field("baz", "Baz")), Field("qux", "Qux")).wrapResponse should be ("Baz => Response[Qux]")
+    Signature("foo", List(Field("baz", "Baz"), Field("qux", "Qux")), Field("zod", "Zod")).wrapResponse should be ("Baz,Qux => Response[Zod]")
```

Comment:
Actually, "quux" comes after "qux". Unless if you are in the DC comics universe, I suppose ;-)

http://en.wikipedia.org/wiki/Metasyntactic_variable